crates/ruff_cli/tests/format.rs

```diff
@@ -813,3 +813,432 @@ fn test_diff_stdin_formatted() {
     ----- stderr -----
     "###);
 }
+
+#[test]
+fn test_notebook_trailing_semicolon() {
+    let fixtures = Path::new("resources").join("test").join("fixtures");
+    let unformatted = fs::read(fixtures.join("trailing_semicolon.ipynb")).unwrap();
+    assert_cmd_snapshot!(Command::new(get_cargo_bin(BIN_NAME))
```

Comment:
I'm not sure if this is the correct place for this test case. We would need to refactor the formatter test suite to include notebook files.